lib/backends/sdc/containers.js

```diff
@@ -546,8 +546,7 @@ function addNetworksToPayload(opts, container, payload, callback) {
     assert.object(opts.config, 'opts.config');
     assert.object(opts.config.napi, 'opts.config.napi');
     assert.object(opts.log, 'opts.log');
-
-    assert.object(opts.config.overlay);
+    assert.object(opts.config.overlay, 'opts.config.overlay');
```

Comment:
I usually assert on callback parameters too with `assert.func(callback, 'callback')`. That sometimes prevents me from spending too much time wondering why a sequence of async operations doesn't complete.